long-running-rest-workitem/src/main/java/org/jbpm/process/longrest/LongRunningRestServiceWorkItemHandler.java

```diff
@@ -0,0 +1,485 @@
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.util.EntityUtils;
+import org.jbpm.process.longrest.util.Mapper;
+import org.jbpm.process.longrest.util.ProcessUtils;
+import org.jbpm.process.longrest.util.Strings;
+import org.jbpm.process.workitem.core.AbstractLogOrThrowWorkItemHandler;
+import org.jbpm.process.workitem.core.util.Wid;
+import org.jbpm.process.workitem.core.util.WidMavenDepends;
+import org.jbpm.process.workitem.core.util.WidParameter;
+import org.jbpm.process.workitem.core.util.WidResult;
+import org.jbpm.process.workitem.core.util.service.WidAction;
+import org.jbpm.process.workitem.core.util.service.WidService;
+import org.kie.api.runtime.KieSession;
+import org.kie.api.runtime.manager.RuntimeManager;
+import org.kie.api.runtime.process.WorkItem;
+import org.kie.api.runtime.process.WorkItemManager;
+import org.kie.api.runtime.process.WorkflowProcessInstance;
+import org.mvel2.MVEL;
+import org.mvel2.ParserContext;
+import org.mvel2.integration.VariableResolverFactory;
+import org.mvel2.integration.impl.MapVariableResolverFactory;
+import org.mvel2.templates.CompiledTemplate;
+import org.mvel2.templates.TemplateRuntime;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.HttpCookie;
+import java.text.MessageFormat;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static org.mvel2.templates.TemplateCompiler.compileTemplate;
+
+
+@Wid(widfile="LongRunningRestService.wid",
+        name="LongRunningRestService",
+        displayName="LongRunningRestService",
+        defaultHandler="mvel: new org.jbpm.contrib.longrest.LongRunningRestServiceWorkItemHandler(runtimeManager)",
+        category="long-running-rest-workitem",
+        documentation = "",
+        parameters={
+                @WidParameter(name="url", required = true),
+                @WidParameter(name="method", required = true),
+                @WidParameter(name="headers", required = false),
+                @WidParameter(name="template", required = false),
+                @WidParameter(name="cancelUrlJsonPointer", required = false),
+                @WidParameter(name="cancelUrlTemplate", required = false)
+        },
+        results={
+                @WidResult(name="responseCode"),
+                @WidResult(name="result"),
+                @WidResult(name="cancelUrl"),
+                @WidResult(name="error")
+        },
+        mavenDepends={
+                @WidMavenDepends(group = "${groupId}", artifact = "${artifactId}", version = "${version}")
+        },
+        serviceInfo = @WidService(category = "REST service", description = "",
+                keywords = "rest,long-running",
+                action = @WidAction(title = "Long running REST service handler ver. ${version}")
```

Comment:
missing
 authinfo = @WidAuth(required = true, params = {"url"}

Reply:
Should I add this line literately or is the "url" a place-holder?